en/guide/message_signing.md

```diff
@@ -0,0 +1,121 @@
+# Message Signing (Authentication)
```

Comment:
I've split out the signing topics because I don't want to silo everything by version - except the stuff to do with "high level overview" and "version handshaking". Hope that makes sense.

Reply:
Makes sense, although v1 doesn't have signing and v3 will have asymmetric signing, so the information here is probably v2 specific.